source/ConversationPanel.h

```diff
@@ -110,9 +110,10 @@ template <class T>
 
 	// The "history" of the conversation up to this point:
 	std::list<Paragraph> text;
-	// The current choices being presented to you:
-	std::list<Paragraph> choices;
+	// The current choices being presented to you, and their indices:
+	std::list<std::pair<Paragraph, int>> choices;
 	int choice;
+	int MapChoice(int n) const;
```

Comment:
Style fix: This line needs moved. This is in the private fields section of the header. The private methods section is further up the file at line 59. Could also use a comment that says what exactly this method does.